### PR TITLE
Add list of JEPs (and status) to README

### PR DESCRIPTION
**Pull request by Zsailer**
I think it might be nice to number JEPs for easy reference. I needed to create a master list of JEPs that have been submitted. I added this list to the front README. They are ordered (and numbered) by their initial submission date. I also *tried* to infer their status.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,26 @@
...
 
-## Index of Accepted JEPs
+## Index of JEPs
+
+Below is a list of JEPs that have been submitted. To view the discussion around each JEP, click on the links below.
+
+1. [Inactive] - [New Notebook Format for improved workflow integration](https://github.com/jupyter/enhancement-proposals/pull/4)
+1. [Inactive] - [Jupyter Extension Generator](https://github.com/jupyter/enhancement-proposals/pull/7)
+1. [Completed] - [Diffing and Merging Notebooks](https://github.com/jupyter/enhancement-proposals/pull/8)
+1. [In progress] - [Kernel Gateway](https://github.com/jupyter/enhancement-proposals/pull/12)
```
Comment (kevin-bates) [inline README.md]: Kernel Gateway should be marked as Completed.

File: README.md [hunk 2]
```diff
@@ -2,9 +2,26 @@
...
+1. [Submitted] - [Kernel Nanny](https://github.com/jupyter/enhancement-proposals/pull/14)
+1. [Withdrawn] - [Layout Namespaces and Discovery](https://github.com/jupyter/enhancement-proposals/pull/15)
+1. [Submitted] - [Notebook Translation and Localization](https://github.com/jupyter/enhancement-proposals/pull/16)
+1. [Completed] - [Declarative Widgets Extension](https://github.com/jupyter/enhancement-proposals/pull/18)
+1. [Completed] - [Dashboards Notebook Extension](https://github.com/jupyter/enhancement-proposals/pull/17)
+1. [Submitted] - [Standalone Jupyter Server](https://github.com/jupyter/enhancement-proposals/pull/28)
+1. [Completed] - [Move Dashboards Deployment Projects from Incubator to Attic](https://github.com/jupyter/enhancement-proposals/pull/22)
+1. [Submitted] - [Jupyter Template as Metadata](https://github.com/jupyter/enhancement-proposals/pull/23)
+1. [Submitted] - [Simplifying Error Reporting in Jupyter Protocol](https://github.com/jupyter/enhancement-proposals/pull/24)
+1. [In progress] - [Enterprise Gateway](https://github.com/jupyter/enhancement-proposals/pull/25)
```
Comment (kevin-bates) [inline README.md]: Enterprise Gateway should be marked as Completed.

File: README.md (hunk 1 above)
Comment (Zsailer) [reply README.md]: 👍  Thanks, Kevin.

File: README.md (hunk 2 above)
Comment (Zsailer) [reply README.md]: 👍  Thanks, Kevin.

File: README.md (hunk 1 above)
Comment (Zsailer) [reply README.md]: Fixed!

File: README.md (hunk 2 above)
Comment (Zsailer) [reply README.md]: Fixed!

**Comment by willingc** [comment]
@Zsailer I'm going to merge and open a PR to iterate this to use the PR number of initial JEP submission as the number. Thanks so much for getting this started.

**Comment by rgbkrk** [comment]
Oooooh nice.

**Pull request merged**